src/Sylius/Bundle/CustomerBundle/DependencyInjection/Configuration.php

```diff
@@ -77,7 +76,6 @@ private function addResourcesSection(ArrayNodeDefinition $node)
                                     ->children()
                                         ->scalarNode('model')->defaultValue(Customer::class)->cannotBeEmpty()->end()
                                         ->scalarNode('interface')->defaultValue(CustomerInterface::class)->cannotBeEmpty()->end()
-                                        ->scalarNode('controller')->defaultValue(CustomerController::class)->cannotBeEmpty()->end()
```

Comment:
Shouldn't this node be left alone, and only `CustomerController` changed to `ResourceController`?